src/Infrastructure/Dashboard/TermMeta/InputTermMetaField.php

```diff
@@ -7,10 +7,10 @@
 class InputTermMetaField implements TermMetaField
 {
     public function __construct(
-        private string $key,
-        private string $title,
-        private string $description = '',
-        private string $type = 'text',
+        protected string $key,
+        protected string $title,
+        protected string $description = '',
+        protected string $type = 'text',
```

Comment:
Why are those properties `protected` now? Is access to those required from elsewhere? And if so, why not use proper access methods instead?

Reply:
Sorry missed your reply for this in #129.